File: Solutions.java/String/0709. To Lower Case.java
/// <summary>
/// 709. To Lower Case
/// https://leetcode.com/problems/to-lower-case
/// 
/// Difficulty Easy
/// Acceptance 83.5%
/// 
/// String
/// </summary>
class Solution {
    public String toLowerCase(String s) {
        var result = new StringBuilder();
        var shift = ('A' - 'a');
        for (var i = 0; i < s.length(); i++){
            var c = s.charAt(i);
            result.append(Character.toLowerCase(c));
        }
        
        return result.toString();
    }
}

File: Solutions.cs/String/0709. To Lower Case.cs
namespace Problem709
{

  /// <summary>
  /// 709. To Lower Case
  /// https://leetcode.com/problems/to-lower-case
  /// 
  /// Difficulty Easy
  /// Acceptance 83.5%
  /// 
  /// String
  /// </summary>
  public class Solution
  {
    public string ToLowerCase(string s)
    {
      var result = new System.Text.StringBuilder();
      for (var i = 0; i < s.Length; i++)
      {
        var c = s[i];
        result.Append(char.ToLower(c));
      }

      return result.ToString();
    }
  }
}
